pkgs/development/libraries/zlog/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, pkgs }:
```

Comment:
It's generally preferred to not pull in the whole "pkgs" set, but instead list each needed dependency.